app/views/owned_tag_sets/_internal_tag_set_fields.html.erb

```diff
@@ -1,5 +1,5 @@
 <% TagSet::TAG_TYPES.each do |tag_type| %>
-  <fieldset class="tagset listbox group">
+  <fieldset class="tagset listbox group"l id="fieldset-<%= tag_type %>">
```

Comment:
It was prep work for more tests, but then I figured out an easier way of doing it, so actually I can remove this change.